index.html

```diff
@@ -49,5 +49,187 @@ <h1 id="name">
             </ul>
         </div>
     </section>
+
+    <section id="work-section">
+
+        <div class="project">
+            <div class="work-img">
+                <img src="/assets/images/projects/Snapshoot Portfolio.png" alt="tonic image">
+            </div>
+            <div class="work-description">
+                <h2 class="work-title">tonic</h2>
+                <ul class="canopy">
+                    <li>canopy</li>
+                    <li>Back End Dev</li>
+                    <li>2015</li>
+                </ul>
+                <p class="work-details">
+                    A daily selection of privately personalized reads; no accounts or sign-ups required.
+                </p>
+                <ul class="work-tools">
+                    <li>HTML</li>
+                    <li>CSS</li>
+                    <li>Javascript</li>
+                </ul>
+                <button class="btn">See Project</button>
+            </div>
+        </div>
```

Comment:
- [ ] **[OPTIONAL]** Awesome work on implementing the previous reviewer's feedback, it's great to see your project section looking so good! However, I have a small suggestion to maintain consistency with the [Figma template](https://www.figma.com/file/l7SqJ3ZfkAKih9sFxvWSR4/Microverse-Student-Project-1?node-id=0-1&t=fpvLyhf07EtGNHXm-0). Please check the text size below the card's name, including **the dot between, and the top and bottom margin**. This will make sure that your project section is even more polished and professional. 💯

### Your design:
![image](https://user-images.githubusercontent.com/107952724/234997324-02e5215d-2b83-4682-b045-2bfcd9a0ac4c.png)

### Figma design:
![image](https://user-images.githubusercontent.com/107952724/234997169-8069baea-e64a-4151-9335-3bc4e308db28.png)